test/kitchen/drivers/ec2-driver.yml

```diff
@@ -15,20 +15,20 @@ provisioner:
 driver:
   name: ec2
   aws_ssh_key_id: <%= ENV['KITCHEN_EC2_SSH_ID'] %>
-  security_group_ids: <%= [ENV['KITCHEN_EC2_SG_IDS']] || ["sg-7fedd80a","sg-46506837"] %>
+  security_group_ids: <%= ENV['KITCHEN_EC2_SG_IDS'] || ["sg-7fedd80a","sg-46506837"] %>
```

Comment:
This seems like a breaking change for the default value from here: https://github.com/DataDog/datadog-agent/blob/8bb415c33dbb9e6d71c5db8ccd8a6d26c488f3b9/.gitlab/kitchen_common/testing.yml#L153

Reply:
Note the default value didn't change.  The format of the list changed.  The defaults are set up to run locally from sandbox.  This way, it parses properly when no variable is set.
The link you put above overrides with the correct SG for the environment when run inthe pipeline.

Reply:
If the env var is not set, this evaluates to `[nil] || ["defaultvalue"]`, which resolves to `[nil]`. I think the default will never be used.